packages/vscode-extension/src/extension.ts

```diff
@@ -36,7 +36,9 @@ export async function activate(context: vscode.ExtensionContext) {
   );
   context.subscriptions.push(createCmd);
 
-  // 1.2 Register the creating command.
+  const debugCmd = vscode.commands.registerCommand("fx-extension.debug", handlers.debugHandler);
```

Comment:
why this format is different to others? e.g.
  const updateCmd = vscode.commands.registerCommand(
    "fx-extension.update",
    handlers.addResourceHandler
  );

Reply:
Because the debug line is shorter than others. I tested locally to make the total length of updateCmd line the same as debugCmd and it changed format to one line.